lib/cWarnings.mli

```diff
@@ -19,3 +19,9 @@ val set_flags : string -> unit
 (** Cleans up a user provided warnings status string, e.g. removing unknown
     warnings (in which case a warning is emitted) or subsumed warnings . *)
 val normalize_flags_string : string -> string
+
+(** [with_warn "-xxx,+yyy..." f ()] calls [f()] after setting the
```

Comment:
commenting myself. The comment is wrong here. There is no need for the () at the end of call to with_warn as it is now.
@Zimmi48 what do you think, should we prefer
```ocaml
with_warn "-foo+bar" f
```
or
```ocaml
with_warn "-foo+bar" f ()
```
to have `f` called?

Reply:
I don't have an opinion on this. In any case, your code looks good to me!